ql/src/test/results/clientpositive/llap/input14_limit.q.out

```diff
@@ -75,19 +75,19 @@ STAGE PLANS:
         Reducer 2 
             Execution mode: vectorized, llap
             Reduce Operator Tree:
-              Select Operator
-                expressions: VALUE._col0 (type: string), VALUE._col1 (type: string)
-                outputColumnNames: _col0, _col1
-                Statistics: Num rows: 500 Data size: 89000 Basic stats: COMPLETE Column stats: COMPLETE
-                Limit
-                  Number of rows: 20
-                  Statistics: Num rows: 20 Data size: 3560 Basic stats: COMPLETE Column stats: COMPLETE
-                  Top N Key Operator
-                    sort order: +
-                    keys: _col0 (type: string)
-                    null sort order: a
+              Limit
```

Comment:
It is worth investigating why we end up with a plan where the parent of a `TNK` is a `Limit` but that can be a follow-up patch since it is out of scope of this patch.

Reply:
I agree, opened HIVE-23957 about it..